### PR TITLE
Update ticdc canal json doc

### PR DESCRIPTION
**Pull request by sdojjy**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/ticdc-canal-json.md [hunk 1]
```diff
@@ -258,10 +258,10 @@ TiCDC 涉及的 Java SQL Type 及其 Code 映射关系如下表所示。
 
 TiCDC 对 Canal-JSON 数据格式的实现，包括 `Update` 类型事件和 `mysqlType` 字段，和官方有些许不同。主要差异见下表。
 
-| 差异点            | TiCDC                  | Canal                                |
-|:----------------|:-------------------------|:-------------------------------------|
-| `Update` 类型事件 | `old` 字段包含所有列数据 | `old` 字段仅包含被修改的列数据          |
-| `mysqlType` 字段  | 对于含有参数的类型，没有类型参数信息         | 对于含有参数的类型，会包含完整的参数信息 |
+| 差异点            | TiCDC                                                                        | Canal                                |
+|:----------------|:-----------------------------------------------------------------------------|:-------------------------------------|
+| `Update` 类型事件 | `old` 字段包含所有列数据, 当 sink 的配置 `only_output_updated_columns` 为 `true` 时只包含被修改的列 | `old` 字段仅包含被修改的列数据          |
```
Comment (qiancai) [inline ticdc/ticdc-canal-json.md:263]:
```suggestion
| `Update` 类型事件 | `old` 字段默认包含所有列的数据。当 sink 参数 `only_output_updated_columns` 设置为 `true` 时，`old` 字段仅包含被修改的列数据 | `old` 字段仅包含被修改的列数据          |
```

File: ticdc/ticdc-open-api-v2.md [hunk 2]
```diff
@@ -319,17 +319,18 @@ curl -X GET http://127.0.0.1:8300/api/v2/health
...
+| `dispatchers`           | 事件分发配置数组。（非必选）                                                                                                       |
+| `encoder_concurrency`   | `INT` 类型。MQ sink 中编码器的线程数。默认值为 `16`。（非必选）                                                                            |
+| `protocol`              | `STRING` 类型，对于 MQ 类的 Sink，可以指定消息的协议格式。目前支持以下协议：`canal-json`、`open-protocol`、`canal`、`avro` 和 `maxwell`。              |
+| `schema_registry`       | `STRING` 类型，schema registry 地址。（非必选）                                                                                 |
+| `terminator`            | `STRING` 类型，换行符，用来分隔两个数据变更事件。默认值为空，表示使用 `"\r\n"` 作为换行符。（非必选）                                                         |
+| `transaction_atomicity` | `STRING` 类型，事务一致性等级。（非必选）                                                                                            |
+| `only_output_updated_columns`             | `BOOLEAN` 类型, 对于 MQ 类型的 Sink 中的 `canal-json`和`open-protocol`，表示是否只向下游同步有内容更新的列 。默认值为 `false`。                        |
```
Comment (qiancai) [inline ticdc/ticdc-open-api-v2.md:333]:
```suggestion
| `only_output_updated_columns`             | `BOOLEAN` 类型，对于 MQ 类型的 Sink 中的 `canal-json` 和 `open-protocol`，表示是否只向下游同步有内容更新的列 。默认值为 `false`。                        |
```

Comment (qiancai) [reply ticdc/ticdc-open-api-v2.md:333]: 是否是  （非必选）

Comment (sdojjy) [reply ticdc/ticdc-open-api-v2.md:333]: 非必选

**Review by asddongmen — APPROVED** [review]

**Review by asddongmen — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

**Pull request merged**